anthos-bm-edge-deployment/README.md

```diff
@@ -117,130 +110,195 @@ gcloud config set compute/zone "${ZONE}"
 # when asked "Create a new key for GSA? [y/N]" type "y" and press
 ./scripts/create-primary-gsa.sh
 ```
+This script will create the JSON key file for the new Google Service Account at
+`./build-artifacts/consumer-edge-gsa.json`. It also sets up KMS keyring and key
+for SSH private key encryption.
+
 ---
 
 ### 2. Provision the GCE instances
-#### 2.1) Configure SSH keys and create the GCE instances where Anthos BareMetal will be installed
 
-> **Note:** _This step can take upto ***2 minutes*** to complete for a setup with $MACHINE_COUNT=3_
+#### 2.1) Create SSH keypair for communication between provisioning machine and target machines
+```sh
+ssh-keygen -N '' -o -a 100 -t ed25519 -f ./build-artifacts/consumer-edge-machine
+```
+
+#### 2.2) Encrypt the SSH private key using Cloud KMS
+```sh
+gcloud kms encrypt \
+  --key gdc-ssh-key \
+  --keyring gdc-ce-keyring \
+  --location global \
+  --plaintext-file build-artifacts/consumer-edge-machine \
+  --ciphertext-file build-artifacts/consumer-edge-machine.encrypted
+```
+
+#### 2.3) Generate the environment configuration file `.envrc` and source it
+Once created inspect the `.envrc` file to ensure that the environment variables
+have been replaced with the correct values.
+
+```sh
+envsubst < templates/envrc-template.sh > .envrc
+source .envrc
+```
 
+#### 2.4) Create the GCE instances where Anthos BareMetal will be installed
+The output of this command will be stored in `./build-artifacts/gce-info`. It
+includes information about the public IP addresses of the GCE VMs and how to SSH
+into them. You can refer to this file later to find the necessary information.
 ```sh
-# just press the return key when asked for a passphrase for the SSH key (i.e. empty string)
-./scripts/cloud/easy-install.sh
+./scripts/cloud/create-cloud-gce-baseline.sh -c "$MACHINE_COUNT" | tee ./build-artifacts/gce-info
 ```
-> **Note:** This script updates your `/etc/hosts` file with the IP addresses of
-> the created GCE VMs. Thus, it will prompt you to provide the **password** to
-> your local workstation
 
-#### 2.2) Test SSH connectivity to the GCE instances
+> **Note:** _This step can take upto ***2 minutes*** to complete for a setup
+> with $MACHINE_COUNT=3_
+
+---
+
+### 3. Install Anthos on bare metal with ansible
+
+This phase involves the creation of the docker image for installation,
+generating the ansible inventory configuration settings and running the ansible
+scripts that installs Anthos on bare metal into the GCE VMs.
```

Comment:
nit: Anoths on Bare Metal

Reply:
Updated all places where this casing as wrong